src/components/SRNEntry.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <section class="w-full flex justify-center items-center my-10">
+    <form class="">
+      <div class="form-group">
+        <label class="text-gray-600 font-semibold text-lg">Enter Student ID</label>
+        <div
+          v-for="(input, index) in studentIDs"
+          :key="`IDInput-${index}`"
+          class="input wrapper flex items-center"
+        >
+          <input
+            v-model="input.studentID"
+            type="number"
```

Comment:
Number? Doesn't the SRN have some letters also? In any case, make this text

Reply:
According to the .xlsx files, it is a 10 digit number only.